Makefile

```diff
@@ -23,5 +23,7 @@ clean:
 	@rm -rf node_modules
 	@echo "Done.\n"
 
+test:
```

Comment:
Wasn't really sure about this, because `make test` is going to be used a lot when debugging/coding, and it's very burdensome to check the dependecy each time (takes some seconds). Maybe adding `make test run` that doesn't check dependencies?